tests/TestData/mutedProblems.json

```diff
@@ -987,6 +987,17 @@
           ]
         }
       ]
+    },
+    {
+      "name": "sodium_crypto_stream_xchacha20_xor_ic",
```

Comment:
Can we just mark this function as removed since 8.2 instead of mute the problem?

Reply:
Not sure how I do this. The problem is that this is a[ new function](https://github.com/php/php-src/blob/1a4401d40aabd9c33fcd122e929f3fbedfbf216f/ext/sodium/libsodium.c#L1585) in 8.2, but reflection returns the function as missing yet.
My plan is to revise this with the next 8.2 release